src/helpers/renderableChildren.js

```diff
@@ -1,5 +1,16 @@
 import React from 'react';
+import { isFragment } from 'react-is';
 
 export default function renderableChildren(childrenProp) {
-  return React.Children.toArray(childrenProp).filter((child) => child === 0 || child);
+  return React.Children.toArray(childrenProp).reduce((children, child) => {
```

Comment:
this could probably be a bit cleaner with https://npmjs.com/array.prototype.flatMap?

Reply:
Good idea. That makes sense to me.

Reply:
Resolved by 8331555